index.html

```diff
@@ -16,13 +17,221 @@
+        <h2>Lift Off</h2>
+        <p>
+          As we got further and further away, it [the Earth] diminished in size. Finally it shrank to the size of a marble, the most beautiful you can imagine. That beautiful, warm, living object looked so fragile, so delicate, that if you touched it with a finger it would fall apart. Seeing this has to change a man.
+
+        </p>
       </div>
+
     </div>
-  </div>
+    <div class="row mt-5 mb-4">
+      <div class="col-12">
+          <div class="row">
+              <div class="col-4">
+                  <img src="https://raw.githubusercontent.com/LambdaSchool/Sprint-Challenge--Web-Tools-And-Frameworks/master/img/blast.jpg" class="img-fluid" alt="Astronaut">
+
+              </div>
+              <div class="col-4">
+                  <img src="https://raw.githubusercontent.com/LambdaSchool/Sprint-Challenge--Web-Tools-And-Frameworks/master/img/launch.jpg" class="img-fluid" alt="Astronaut">
+
+              </div>
+              <div class="col-4">
+                  <img src="https://raw.githubusercontent.com/LambdaSchool/Sprint-Challenge--Web-Tools-And-Frameworks/master/img/sky.jpg" class="img-fluid" alt="Astronaut">
+
+              </div>
+            </div>
+      </div>
+    </div>
+  
+    <div class="row">
+      <div class="col-sm-12 mt-5 mb-5">
+        <h2>Can We Visit Planets?</h2>
+        <p>
+          Never in all their history have men been able truly to conceive of the world as one: a single sphere, a globe, having the qualities of a globe, a round earth in which all the directions eventually meet.
+        </p> 
+          <p> 
+          which there is no center because every point, or none, is center — an equal earth which all men occupy as equals. The airman's earth, if free men make it.
+        </p>
+      </div>
+
+    </div>
+    <!--start table-->
+    <div class="row">
+      <div class="col-sm-12 mt-5 mb-5">
+        <table class="table table-striped">
+          <thead>
+            <tr>
+              <th scope="col">Planet</th>
+              <th scope="col">Distance</th>
+              <th scope="col">Survivability</th>
+              <th scope="col">How Long To Travel</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <th scope="row">Mars</th>
+              <td>54.6 million km</td>
+              <td>Maybe</td>
+              <td>300 days</td>
+            </tr>
+            <tr>
+              <th scope="row">Jupiter</th>
+              <td>588 million km</td>
+              <td>No</td>
+              <td>Depends On Orbit (3-6 Years)</td>
+            </tr>
+            <tr>
+              <th scope="row">Saturn</th>
+              <td>1.2 billion km</td>
+              <td>No</td>
+              <td>Depends On Orbit (3-6 Years)</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div> <!--endTable-->
+    <div class="row mb-5 d-flex justify-content-center">
+      <div class="col-4 mt-5">
```

Comment:
Take a look at this div container unless you intended it to be that way. The text and image are not aligned to the full container but rather in the center. This is just a recommendation because it looks great either way!